documentation/migration-guides/Az.6.0.0-migration-guide.md

```diff
@@ -0,0 +1,378 @@
+# Migration Guide for Az 6.0.0
+- [Migration Guide for Az 6.0.0](#migration-guide-for-az-600)
+	- [Az.Accounts](#azaccounts)
+		- [`Connect-AzAccount`](#connect-azaccount)
+	- [Az.ContainerInstance](#azcontainerinstance)
+		- [`New-AzContainerGroup`](#new-azcontainergroup)
+		- [`Remove-AzContainerGroup`](#remove-azcontainergroup)
+		- [`Get-AzContainerGroup`](#get-azcontainergroup)
+		- [`Get-AzContainerInstanceLog`](#get-azcontainerinstancelog)
+	- [Az.DesktopVirtualization](#azdesktopvirtualization)
+		- [`New-AzWvdHostPool`](#new-azwvdhostpool)
+		- [`Expand-AzWvdMsixImage`](#expand-azwvdmsiximage)
+		- [`New-AzWvdMsixPackage`](#new-azwvdmsixpackage)
+		- [`Update-AzWvdHostPool`](#update-azwvdhostpool)
+	- [Az.StreamAnalytics](#azstreamanalytics)
+		- [`Get-AzStreamAnalyticsDefaultFunctionDefinition`](#get-azstreamanalyticsdefaultfunctiondefinition)
+		- [`New-AzStreamAnalyticsJob`](#new-azstreamanalyticsjob)
+		- [`New-AzStreamAnalyticsTransformation`](#new-azstreamanalyticstransformation)
+	- [Az.RecoveryServices](#azrecoveryservices)
+		- [`Set-AzRecoveryServicesBackupProperty`](#set-azrecoveryservicesbackupproperty)
+		- [`Get-AzRecoveryServicesBackupJobDetail`](#get-azrecoveryservicesbackupjobdetail)
+	- [Az.Storage](#azstorage)
+		- [`Remove-AzRmStorageShare`](#remove-azrmstorageshare)
+	- [Az.ServiceFabric](#azservicefabric)
+		- [`Add-AzServiceFabricClusterCertificate`](#add-azservicefabricclustercertificate)
+		- [`Get-AzServiceFabricManagedClusterService`](#get-azservicefabricmanagedclusterservice)
+		- [`New-AzServiceFabricManagedCluster`](#new-azservicefabricmanagedcluster)
+		- [`New-AzServiceFabricManagedClusterService`](#new-azservicefabricmanagedclusterservice)
+		- [`Remove-AzServiceFabricClusterCertificate`](#remove-azservicefabricclustercertificate)
+		- [`Remove-AzServiceFabricManagedClusterService`](#remove-azservicefabricmanagedclusterservice)
+		- [`Set-AzServiceFabricManagedCluster`](#set-azservicefabricmanagedcluster)
+		- [`Set-AzServiceFabricManagedClusterService`](#set-azservicefabricmanagedclusterservice)
```

Comment:
We need to add a section about the PowerShell supportability.

### Supported versions of PowerShell

Due to [CVE-2021-26701](https://msrc.microsoft.com/update-guide/en-us/vulnerability/CVE-2021-26701) Az 6 is only supported on the following platforms:
- PowerShell 7.1: version 7.1.3 or above
- PowerShell 7.0: version 7.0.6 or above
- Windows PowerShell 5.1 

For further details, refer to the [Azure PowerShell support lifecycle](https://aka.ms/lifecycle)

Reply:
Added.